deps.sh

```diff
@@ -118,11 +118,12 @@ function install_cramfstools
 
 function install_ubireader
 {
+    install_pip_package python-lzo
     git clone https://github.com/jrspruitt/ubi_reader
     # Some UBIFS extraction breaks after this commit, due to "Added fatal error check if UBI block extends beyond file size"
     # (see this commit: https://github.com/jrspruitt/ubi_reader/commit/af678a5234dc891e8721ec985b1a6e74c77620b6)
```

Comment:
Maybe remove this comment too?

Reply:
remove why ?

Reply:
The bug in the comment should be fixed [here](https://github.com/jrspruitt/ubi_reader/commit/63105bf3c0b1618f029da65d73e0ed7a0874e695).